src/docs/webassembly-opcode.md

```diff
@@ -0,0 +1,430 @@
+---
+title: 'WebAssembly - adding a new opcode'
+---
+
+[WebAssembly](https://webassembly.org) (Wasm) is a binary instruction format for a stack-based virtual machine. This tutorial walks the reader through implementing a new WebAssembly instruction in v8.
+
+WebAssembly is implement in v8 in 3 ways:
```

Comment:
implemented

Reply:
nit: `s/v8/V8/` (multiple occurrences throughout the document)

Reply:
@mathiasbynens Note you can use Github suggestions for these sort of fixes, should make it a bit easier:

```suggestion
WebAssembly is implemented in V8 in 3 ways:
```